tensorflow/tools/pip_package/setup.py

```diff
@@ -114,7 +114,7 @@ def standard_or_nightly(standard, nightly):
     # current release version. These also usually have "alpha" or "dev" in their
     # version name.
     # These are all updated during the TF release process.
-    standard_or_nightly('tensorboard >= 2.12, < 2.13',
+    standard_or_nightly('tensorboard ~= 2.13, < 2.14',
```

Comment:
This is weird, given that `~2.13` actually means `>= 2.13, <3`. Should this be either `~= 2.13` or `>= 2.13, < 2.14`?